### PR TITLE
Bug/manage program users tooltip

### PR DESCRIPTION
**Pull request by hlminh2000**
**Description of changes**

- make Tooltip unmount html when hidden
- adds style to support unmounting (needs to be global)
  - enables importing css files
- make actions tooltip appear on left

**Type of Change**

- [x] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: uikit/Tooltip/index.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* 
+  We need this file applied globally for react-tippy to work, hence the css file
+ */
+.tippy-popper .leave {
```
Comment (d8660091) [inline uikit/Tooltip/index.css]: Can we put them in the style block of ApplicationRoot?

Comment (hlminh2000) [reply uikit/Tooltip/index.css]: it seems more of a concern for the Tooltip component than the application so I figured it makes more sense here :P do you think we prefer application root? 

If there was another way to make the style apply globally without using css imports while keeping the definition scoped in this component context I'd prefer that too :)

**Comment by hlminh2000** [comment]
@joneubank no more hack to monitor thanks to @d8660091 :D

**Comment by d8660091** [comment]
Great job!

**Pull request merged**